okta-hosted-login/README.md

```diff
@@ -1,30 +1,65 @@
 
 ## Prerequisites
 
-Before running this sample, you will need the following:
+Before running this example, you will need the following:
 
 * An Okta Developer Account, you can sign up for one at https://developer.okta.com/signup/.
-* An Okta Application, configured for Web mode. This is done from the Okta Developer Console and you can find instructions [here][OIDC Web Application Setup Instructions].  When following the wizard, use the default properties.  They are designed to work with our sample applications.
-
+* An Okta Application, configured for Web mode. This is done from the [Okta Developer Console] and you can find instructions [here][OIDC Web Application Setup Instructions].  When following the wizard, use the default properties.  They are designed to work with our sample applications.
 
 ## Running This Example
 
-Clone this repo and replace the okta configuration placeholders in the `appsettings.json` with your configuration values from the Okta Developer Console. 
+### Clone this repository
+
+```git clone https://github.com/okta/samples-aspnetcore.git```
+
+### Run the web application
+
+Run the example with your preferred tool and write down the port of your web application to configure Okta afterwards.
+
+> **NOTE:** This sample is using ASP.NET Core 2.2 which enforces HTTPS. This is a recommended practice for web applications. Check out [Enforce HTTPS in ASP.NET Core] for more details.
+
+#### Run the web application from Visual Studio
+
+If run this project in Visual Studio it will start the web application on ports 5000 for HTTP and 44314 for HTTPS. You can change this configuration in the `launchSettings.json`. 
+
+#### Run the web application from dotnet CLI
+
+If you run this project via the dotnet CLI it will start the web application on ports 5000 for HTTP and 5001 for HTTPS. You can change this configuration in the `launchSettings.json`. 
+
+#### Trust the local dev certificate if necessary
+
+If you’ve never run an ASP.NET Core 2.x application before, you may notice a strange error page come up warning you that the site is potentially unsafe.
+This is because ASP.NET Core creates an HTTPS development certificate for you as part of the first-run experience, but it still needs to be trusted. You can ignore the warning by clicking on Advanced and telling the browser that it’s okay to visit this site even though there is no certificate for it. Or you can trust the certificate to get rid of this warning, check out [Configuring HTTPS in ASP.NET Core across different platforms] for more details.
+
+### Add the correct configuration to the Okta Developer Console (including the port you just found in [Run the web application](#run-the-web-application))
+
+Go to your [Okta Developer Console] and update the following parameters in your Okta Web Application configuration:
+    * **Login redirect URIs**
```

Comment:
Add an example here of what each of these redirect URIs will look like, like:

**Login redirect URI** - for example, `https://localhost:5000/signin/callback`
**Logout redirect URI** - for example, `https://localhost:5000/signout/callback`